docs/rule/builtin-component-arguments.md

```diff
@@ -0,0 +1,48 @@
+# builtin-component-arguments
+
+The builtin `Input` component has several arguments that match attributes
+of the lower-case `input` HTML element. These arguments should be set via e.g.
+`@type`, instead of `type`, but it is easy to forget and can cause subtle
+issues.
+
+This rule warns about `Input` component invocations that use `type`, `value` or
+`checked` attributes, instead of arguments.
```

Comment:
```suggestion
This rule warns about `Input` component invocations that use the following attributes instead of arguments:

* `checked`
* `type`
* `value`
```